assets/css/base/_type-scale.scss

```diff
@@ -0,0 +1,74 @@
+@use "sass:map";
+
+/// Create new/different scales by editing the config
+/// This is the single source of truth
+$fluid-config: (
+  // Typescale ratio for calculating steps
+    typescale-ratio: 1.25,
+  // base font size to use for --step0
+    base-fontsize: 1rem,
+  // adjustment added to base-fontsize in --step0 for scaling
+    viewport-scale: 0.5vw,
+  // number of positive steps (--step1, --step2 etc)
+    // and number of negative steps (--step-1, --step-2 etc)
```

Comment:
These names are not really clear. Hyphenating is common to separate words, especially in CSS where we don’t use camel case. Can you change these two be either `--step-pos-1` and `--step-neg-1` or for positive use `--step-1` (as this is what we mostly use) and only namespace the negative variables, e.g. `--step-minus-1`

Reply:
I agree. Will update